frontend/src/permissions.js

```diff
@@ -64,11 +64,15 @@ export const getRegionWithReadWrite = (user) => {
  * @param {*} user - user object
  * @returns {boolean} - True if the user has re/write access for a region, false otherwise
  */
-export const hasReadWrite = (user) => {
+const hasReadWrite = (user) => {
   const { permissions } = user;
   return permissions && permissions.find(
     (p) => p.scopeId === SCOPE_IDS.READ_WRITE_ACTIVITY_REPORTS,
   ) !== undefined;
 };
 
-export default isAdmin;
+export {
+  isAdmin as default,
+  getRegionWithReadWrite,
+  hasReadWrite,
+};
```

Comment:
I love consolidating these here, and I didn't know that you could specify `default` as part of `export { xxx as yyy }`. Neat trick